build.sbt

```diff
@@ -74,6 +74,18 @@ lazy val root = project
     federation
   )
 
+lazy val macros = project
+  .in(file("macros"))
```

Comment:
Why is it needed?

Reply:
Macros cannot be invoken in same compilation unit where they defined.
`Schema.gen` used in `caliban.introspection.Introspector`

Reply:
Makes sense 👍